### PR TITLE
Automated stubbed test case in repository api file upload feature.

### PR DESCRIPTION
**Pull request by adarshdubey-star**
**Test result - 6.10**

```
pytest tests/foreman/api/test_repository.py -k test_positive_upload_file_to_file_repo
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.8.6, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/addubey/work/robottelo, configfile: pyproject.toml
plugins: services-2.2.1, mock-3.6.1, reportportal-5.0.8, forked-1.3.0, ibutsu-1.16, cov-3.0.0, xdist-2.4.0
collected 187 items / 186 deselected / 1 selected                                                                                                                                                                 

tests/foreman/api/test_repository.py .                                                                                                                                                                     [100%]

================================================================================= 1 passed, 186 deselected,in 43.74s ==================================================================================


```

### CODE REVIEW

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_upload_file_to_file_repo' tests/foreman/api/test_repository.py

**Review by mshriver — APPROVED** [review]

**Review by ColeHiggins2 — APPROVED** [review]
ACK

File: tests/foreman/api/test_repository.py [hunk 1]
```diff
@@ -35,6 +35,9 @@
 from robottelo.api.utils import upload_manifest
 from robottelo.config import settings
 from robottelo.constants import repos as repo_constants
+from robottelo.constants import RPM_TO_UPLOAD
+from robottelo.constants.repos import CUSTOM_FILE_REPO
```
Comment (mshriver) [inline tests/foreman/api/test_repository.py:39]: Missed this earlier. `from robottelo.constants import repos as repo_constants` is already here, you can already reference this with `repo_constants.CUSTOM_FILE_REPO`. 

`from robottelo import constants` is also already defined above. So you can reference `constants.RPM_TO_UPLOAD`.

Both of these imports are redundant.

Comment (adarshdubey-star) [reply tests/foreman/api/test_repository.py]: Noted

**Review by mshriver — APPROVED** [review]

**Pull request merged**